jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/filters/basic/BasicFiltersPresenter.java

```diff
@@ -74,6 +75,7 @@ public void onClearAllActiveFiltersEvent(@Observes ClearAllActiveFiltersEvent ev
 
     protected void addSearchFilter(final ActiveFilterItem filter,
                                    final ColumnFilter columnFilter) {
+        ((CoreFunctionFilter)columnFilter).setLabelValue(filter.getLabelValue());
```

Comment:
unchecked cast, in class below there's a checked cast. Isn't it necessary here too?

Reply:
LabelValue is CoreFuntionFilter's field, so need it.
https://github.com/kiegroup/kie-soup/pull/99/files#diff-796480d638431e6b8cbb1c6bf162a2cfR27

Reply:
I don't question the casting itself, but rather that we're not checking if the columnFilter can be cast to CoreFunctionFilter.
Like you did in file below:

```
if (columnFilter instanceof CoreFunctionFilter) {
    labelValue = ((CoreFunctionFilter) columnFilter).getLabelValue();
```

Reply:
ColumnFilter have three sun class LogicalExprFilter and CustomFunctionFilter and CoreFunctionFilter, it will avoid class cast exception.

Reply:
@bxf12315 agreed, but you don't have the check here.

Reply:
@jstastny-cz where is better? My change code in here, so I think I need to check it in here.

Reply:
I'd add the checks to both places, to prevent classcastexception as you mentioned earlier.

Reply:
Where is it?， you have added?

Reply:
done

Reply:
I would add it, I haven't, that was a proposal.

Reply:
no no, you are right. This code has been more than a month old, I am a bit confused.